### ISSUE TITLE
Keep mouse pointer away after hinting

### ISSUE THREAD

**Issue by sudo-nice** [qutebrowser/qutebrowser#1197]
I realize, hints are triggered by mouse clicking by design. Sometimes it's OK, but sometimes mouse pointer left in the middle of the browser's window (even with `hide-mouse-cursor = true`) starts interact with the stuff, and it's annoying. I often use `xdootol mousemove` to get rid of that annoying pointer, but I'm loosing that war currently. Since I don't need mouse actions with the qutebrowser at all, is there any cure?

**Comment by The-Compiler** [comment]
Just to be sure we're talking about the same thing: e.g. on http://www.heise.de/ I use hints to select one of the "Topthemen" links via hints, and then when I scroll down, the links of the articles are hovered and thus the hover text is displayed in the statusbar - is this the same issue you're seeing?

**Comment by sudo-nice** [comment]
I think, we are talking about the same issue, howhever I can't reproduce it on http://www.heise.de but I can do it with google images site, e.g.

**Comment by lahwaacz** [comment]
If there is a general way to completely move the cursor away instead of just hiding it, I'd expect it to be added to [unclutter-xfixes](https://github.com/Airblader/unclutter-xfixes). Perhaps reporting there is worth a shot?

**Comment by sudo-nice** [comment]
Another example of the mentioned behaviour: ![screenshot](http://i.imgur.com/Om6MvMg.png)

**Comment by sudo-nice** [comment]
Looks like using uclutter this way `unclutter -notclass qutebrowser` helps to avoid the mouse cursor from interaction with the page's content. I only need to move the pointer to the corner of the qutebrowser's window. I beleive I'm done with the issue.

**Comment by The-Compiler** [comment]
This is fixed nowadays with normal clicking, but still an issue with e.g. `;h`.

**Comment by sudo-nice** [comment]
Seems nice, thanks a lot!

**Comment by The-Compiler** [comment]
I still want to keep this open as long as it's still a problem with some hinting modes :wink:

**Comment by daniel-Jones** [comment]
Using the commit 3b957c5f2 (webkit) default hinting is causing the same mouse behavior as mentioned by sudo-nice.

**Comment by amacfie** [comment]
Workaround: do `;h` on another element like you would with the mouse

**Comment by RossSmyth** [comment]
Yeah this is a bit annoying when reading Wikipedia articles cause I'll do a hint hover on a link to see a breif summary of what it is, then if I scroll down it will hover on another link.
